gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -315,6 +315,7 @@ main (int argc, char *argv[])
   /* Upstream has "existing user" mode for new user accounts. In Endless, we
      skip straight to the welcome tutorial instead. */
   if (get_mode () == GIS_DRIVER_MODE_EXISTING_USER) {
+    setlocale (LC_ALL, "");
```

Comment:
If you want, but I'm not sure that's best. For an upstream patch, we'd of course want a much more thorough change, but for Endless, don't you think it's probably best to quit early in main, before running GisDriver, since we don't need any of its functionality and to minimize our delta?

Also, since we don't use GTK+ at all, I don't see why we should move this code beneath GTK+ initialization. It seems like overkill to initialize GTK+ just to get it to call setlocale() for us. :)